### PR TITLE
rewrite src/modules/backuper/DropboxBackuper.js, LocalBackuper.js with ES6 syntax

### PR DESCRIPTION
**Pull request by neighborhood999**

### CODE REVIEW

**Comment by EragonJ** [comment]
@neighborhood999 can you break Backuper into two several commits ? Thanks :)

**Comment by neighborhood999** [comment]
@EragonJ done!

File: src/modules/backuper/DropboxBackuper.js [hunk 1]
```diff
@@ -1,219 +1,218 @@
...
+const BrowserWindow = Remote.BrowserWindow;
+
+class DropboxBackuper {
+  constructor() {
+    this._dropboxAccessToken = '';
+    this._dropbox = null;
   }
 
-  var folderName = options.folderName;
+  backup(datas, opitons) {
```
Comment (EragonJ) [inline src/modules/backuper/DropboxBackuper.js]: `options` !

**Comment by EragonJ** [comment]
Never mind, I helped to change that :) already merged !

**Comment by EragonJ** [comment]
With this patch, it's time to upgrade dropbox API to v2.0 !

**Comment by neighborhood999** [comment]
@EragonJ thanks XD

**Pull request merged**